lib/internal/Magento/Framework/Message/Manager.php

```diff
@@ -79,7 +79,7 @@ public function __construct(
         Event\ManagerInterface $eventManager,
         LoggerInterface $logger,
         $defaultGroup = self::DEFAULT_GROUP,
-        ?ExceptionMessageFactoryInterface $exceptionMessageFactory = null
+        ExceptionMessageFactoryInterface $exceptionMessageFactory = null
```

Comment:
Let's revert this change:
```suggestion
        ?ExceptionMessageFactoryInterface $exceptionMessageFactory = null
```